.gitignore

```diff
@@ -8,4 +8,6 @@
 release
 .jekyll-cache
 .jekyll-metadata
-NOTES
\ No newline at end of file
+NOTES
+
+.direnv/
```

Comment:
This change also seems unnecessary. Please revert.

Reply:
No it isn't. If you're using [`direnv`](https://direnv.net/) with flakes, then this directory will be created which isn't required for every user.